src/NuGet/Microsoft.NETCore.Compilers/tools/bincore/RunCsc

```diff
@@ -1,14 +0,0 @@
-#!/bin/sh
```

Comment:
Were this file and related ones below included in packages before? If so, does removing them risk breaking some consumers of those packages?